projects/core/src/util/date-picker-formatter.service.ts

```diff
@@ -23,4 +23,9 @@ export class DatePickerFormatterService {
       return date;
     }
   }
+
+  toISOString(value: string): string {
+    const dateTime = '00:00:00';
+    return value ? this.toNative(value) + 'T' + dateTime + 'Z' : null;
```

Comment:
I have mentioned that the date logic is wrong in replenishment, which is why we need to use re-use the service that came from my-company, however, we need to check if the logic is fine as well and fix or create things accordingly.

For example of why this logic doesn't work, try placing a replenishment for January 1st 2021, and you will see that the returned date is December 31st 2020, which is wrong.

Moreover, `toModel` function is doing the same thing as this imported function from serializer, when a user chooses a date from the date picker.

Reply:
To shed some light on what is happening with the DatePicker, it uses the native `datetime-local` input element which will automatically append the timezone information (eg. CEST +2) when the datetime value is submitted in a request to the backend.

The backend cannot store this timezone information and will store the time in UTC-0 and adding/subtracting timezone offset hours to the final value stored.

If we display this date again from a backend response and put the value through a date pipe such at `cxDate`, the local timezone is used to convert the date value from the backend to the time relevant to the client (ie. local time offset value is displayed).

PS. I recall an issue with daylight savings time where the wrong offset is used if you are currently in/using a local timezone in daylight savings and the date you are setting to is not. Incorrect dates can be displayed or submitted to the backend by an hour. This was ignored at the time as an hour's discrepancy does not make the component unusable but I suspect that is what's happening with this issue.

Reply:
@bgambocjaviniar @Zeyber  

Thanks for the input. I will create a fresh PR and add detailed comments and findings.